docs/src/man/convex_sets.md

```diff
@@ -0,0 +1,269 @@
+# Introduction to Convex Sets
+
+In this section we give a short overview about convex sets supported in this
+library and the corresponding lazy set calculus using support functions.
+
+```@contents
+Pages = ["convex_sets.md"]
+Depth = 4
+```
+
+```@setup convex_sets
+using LazySets, Plots, Plots.PlotMeasures, LaTeXStrings
+
+c1 = "blue"
+c2 = "green"
+c3 = "red"
+c4 = "orange"
+
+function plot_raw()
+    plot(tickfont=font(15, "Times"), aspectratio=1, size=(300, 300), leg=false)
+end
+
+function plotX(X, ε, modify, color; linecolor="black")
+    if !modify
+        plot_raw()
+    end
+    plot!(X, ε, color=color, alpha=0.7, linecolor=linecolor)
+end
+
+function plotX(X, modify, color; linecolor="black")
+    if !modify
+        plot_raw()
+    end
+    plot!(X, color=color, alpha=0.7, linecolor=linecolor)
+end
+
+function plot1(X, modify::Bool=false; linecolor="black")
+    plotX(X, modify, c1, linecolor=linecolor)
+end
+
+function plot2(X, modify::Bool=false; linecolor="black")
+    plotX(X, modify, c2, linecolor=linecolor)
+end
+
+function plot3(X, modify::Bool=false; linecolor="black")
+    plotX(X, modify, c3, linecolor=linecolor)
+end
+
+function plot1(X, ε::Float64, modify::Bool=false; linecolor="black")
+    plotX(X, ε, modify, c1, linecolor=linecolor)
+end
+
+function plot2(X, ε::Float64, modify::Bool=false; linecolor="black")
+    plotX(X, ε, modify, c2, linecolor=linecolor)
+end
+
+function plot3(X, ε::Float64, modify::Bool=false; linecolor="black")
+    plotX(X, ε, modify, c3, linecolor=linecolor)
+end
+
+function plot_hyperplane!(x, y, linecolor)
+    plot!(x, y, linecolor=linecolor, arrow=:arrow, width=8)
+    plot!([x[2], x[1]], [y[2], y[1]], linecolor=linecolor, arrow=:arrow, width=8)
+end
+```
+
+## Basic convex sets
+
+A set ``X ⊆ \mathbb{R}^n`` is convex if for any two points ``x``, ``y`` in ``X``
+the whole line segment connecting ``x`` and ``y`` is also contained in ``X``.
+Formally:
+
+```math
+    X = \{λ · x + (1 − λ) · y \mid x, y ∈ X, λ ∈ [0, 1] ⊆ \mathbb{R}\}
+```
+
+Below we show an example of a convex set and a non-convex set.
+
```

Comment:
i would say here that `VPolygon` represents the convex hull of the given vertices, and add a sentence of about `HPolygon` too (in both cases with links to the types' docs / wikipedia)